src/comment_parser.rs

```diff
@@ -73,13 +73,19 @@ pub enum CommentItem {
     BibTag(Span),
 }
 
+enum UnderscoreMode {
+    Subscript,
+    Italic,
+    Underscore,
+}
+
 /// Returns true if this is a character that is escaped in [`CommentItem::Text`],
 /// [`CommentItem::Label`] and [`CommentItem::Url`] fields,
 /// meaning that doubled occurrences are turned into single occurrences.
 #[inline]
 #[must_use]
 pub const fn is_text_escape(c: u8) -> bool {
-    matches!(c, b'`' | b'[' | b'~')
+    matches!(c, b'`' | b'[' | b'~' | b'_')
 }
```

Comment:
@benjub This is where we state that double occurrences of underscores are turned into single occurrences.

Reply:
Thank you!